test/storage/StorageTest.cc

```diff
@@ -12,32 +11,65 @@
 
 INITIALIZE_EASYLOGGINGPP
 
-static const int NUM_DATANODES = 3;
-
 using asio::ip::tcp;
 using client_namenode_translator::ClientNamenodeTranslator;
 
+static const int NUM_DATANODES = 3;
+
+int32_t xferPort = 50010;
+int32_t ipcPort = 50020;
+int maxDatanodeId = 0;
+// Use minDatanodId++ when you want to kill a datanode.
+int minDatanodeId = 0;
+uint16_t nextPort = 5351;
+
+static inline void initializeDatanodes(int numDatanodes) {
```

Comment:
Is this code duplicated from the file ReadWriteTest.cc? Could we factor out into a common file?

Reply:
done